compiler/symbol.go

```diff
@@ -191,6 +200,10 @@ func (info *functionInfo) parsePragmas(f *ssa.Function) {
 		return
 	}
 	if decl, ok := f.Syntax().(*ast.FuncDecl); ok && decl.Doc != nil {
+
+		// Our importName for a wasm module (if we are compiling to wasm), or llvm link name
+		var importName string = ""
```

Comment:
Just a nitpick: this can be declared slightly shorter as follows:

```go
importName := ""
```

or like this:

```go
var importName string
```